.env.example

```
@@ -1 +1,73 @@
+
+# jwt
+JWT_SECRET=
+JWT_ACCESS_TOKEN_EXPIRATION=
+JWT_REFRESH_TOKEN_EXPIRATION=
+
+# Kakao API
+KAKAO_API_KEY=
+
+# slack
+SLACK_USER_TOKEN=
+SLACK_BASE_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SLACK_BASE_URL key should go before the SLACK_USER_TOKEN key

(UnorderedKey)